<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_run</class>
 <widget class="QWidget" name="Mode_run">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>481</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Attitude</string>
   </property>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RPY:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxRoll">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxPitch">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxYaw">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_79">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_tilt_cal">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NR</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_change_mode_run">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode Run</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_12">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VX:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_vx">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VY:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>OMEGA:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBoxVY">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtb_pidt_w">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_control_enable">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Disabled</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_log">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Log</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Control</string>
   </property>
   <widget class="QPlainTextEdit" name="pteControl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
   <property name="title">
    <string>Measurement</string>
   </property>
   <widget class="QPlainTextEdit" name="pteMeasurement">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
